### PR TITLE
port: house

### PR DESCRIPTION
**Pull request by glennj**
This will be the first of 3 song generators (akin to beer song): next will be proverb and twelve-days.

[canonical-data.json](https://github.com/exercism/problem-specifications/blob/master/exercises/house/canonical-data.json)

---

Reviewer Resources:

[Track Policies](https://github.com/exercism/bash/blob/master/POLICIES.md)

### CODE REVIEW

File: exercises/house/house_test.sh [hunk 1]
```diff
@@ -0,0 +1,369 @@
...
+    run bash house.sh 1 12
+    [[ $status -eq 0 ]]
+    [[ $output == "$expected" ]]
+}
+
+
+@test "invalid verse 1" {
+    skip
+    run bash house.sh 0 12
+    [[ $status -ne 0 ]]
```
Comment (guygastineau) [inline exercises/house/house_test.sh]: Do you think these invalid cases should return 1?

Comment (glennj) [reply exercises/house/house_test.sh]: `-ne`

Comment (guygastineau) [reply exercises/house/house_test.sh]: Thank you. I don't know why I keep missing that.

**Comment by guygastineau** [comment]
Thank you.

**Pull request merged**